src/styles/blocks/stopwatch.scss

```
@@ -0,0 +1,44 @@
+.stopwatch {
+  position: relative;
+  margin-bottom: 20px;
+  border: 3px solid black;
+  @include stopwatch-size();
+
+  &__center-point {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+
+    width: 8%;
+    height: 8%;
+    border-radius: 50%;
+    border: 2px solid red;
+  }
+
+  &__minute-hand {
+    @extend %hands-border;
+    @include hands-position;
+
+    margin: auto;
+    height: 45%;
+    width: 0;
+    border-color: red;
+
+    animation: stopwatch 60s linear 1s infinite;
+    transform-origin: bottom center;
+  }
+
+  &__second-hand {
+    @extend %hands-border;
+    @include hands-position;
+
+    margin: auto;
+    height: 49%;
+    width: 0;
+    border-color: black;
+
+    animation: stopwatch 1s linear 1s infinite;
```

```suggestion
    animation: stopwatch 60s linear 1s infinite;
```
1s is the time of a full rotation of the hand. To simulate a stop-watch  here should be seconds

---

> 1s is the time of a full rotation of the hand. To simulate a stop-watch here should be seconds

Sorry, but I still confused.
I have two hands: one with 1s duration other with 60s. They should be 60s each?